e2e/tests/121-Portal/ViewPaProfilePage/OpenProfilePage.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect } from '@playwright/test';
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { importRegistrationsCSV } from '@121-service/test/helpers/registration.helper';
+import { getAccessToken, resetDB } from '@121-service/test/helpers/utility.helper';
+
+test.beforeEach(async ({ page }) => {
+  // Reset the DB to the required state
+  const response = await resetDB(SeedScript.nlrcMultiple);
+  expect(response.status).toBe(202);
+  // Upload registration from the file
+  const programIdOcw = 3;
+  const accessToken = await getAccessToken();
+  await importRegistrationsCSV(
+    programIdOcw,
+    './fixtures/test-registrations-OCW.csv',
```

Comment:
We could also import this from the `@121-service`-here maybe?

Reply:
It didn't work.